Makefile.include

```diff
@@ -280,6 +280,16 @@ objdump:
 		exit 1; }
 	$(PREFIX)objdump -S -D -h $(ELFFILE) | less
 
+# Generate an XML file containing all macro definitions and include paths for
+# use in Eclipse CDT
+.PHONY: eclipsesym eclipsesym.xml $(CURDIR)/eclipsesym.xml
+eclipsesym: $(CURDIR)/eclipsesym.xml
+eclipsesym.xml: $(CURDIR)/eclipsesym.xml
+
+$(CURDIR)/eclipsesym.xml:
+	$(AD)echo $(CC) $(CFLAGS) $(INCLUDES) | \
```

Comment:
I wanted to be able to run the script easily outside of make as well, so I figured stdin->stdout was the most convenient method.